pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -613,14 +613,20 @@ func GenerateService(cfg echo.Config) (string, error) {
 
 var VMImages = map[echo.VMDistro]string{
 	echo.UbuntuXenial: "app_sidecar_ubuntu_xenial",
-	echo.UbuntuFocal:  "app_sidecar_ubuntu_focal",
-	echo.UbuntuBionic: "app_sidecar_ubuntu_bionic",
-	echo.Debian9:      "app_sidecar_debian_9",
-	echo.Debian10:     "app_sidecar_debian_10",
+	echo.UbuntuJammy:  "app_sidecar_ubuntu_jammy",
+	echo.Debian11:     "app_sidecar_debian_11",
 	echo.Centos7:      "app_sidecar_centos_7",
-	echo.Centos8:      "app_sidecar_centos_8",
+	echo.Rockylinux8:  "app_sidecar_rockylinux_8",
 }
 
+var RevVMImages = func() map[string]echo.VMDistro {
```

Comment:
I think previously we just always tested DefaultVMDistro repeatedly... fixing this